lib/microos.pm

```diff
@@ -22,7 +22,7 @@ our @EXPORT = qw(microos_reboot microos_login);
 
 # Assert login prompt and login as root
 sub microos_login {
-    assert_screen 'linux-login-microos', 150;
+    assert_screen('linux-login-microos', 150) unless match_has_tag('linux-login-microos');
```

Comment:
Is this change necessary?

This might break if `microos_login` is called after a tty switch or so, but currently that's not the case

Reply:
I'm trying to work on staging images for SUSE MicroOS, there are some recent changes that need needling. So, I put this in WIP until I find some stability in those images and also will run VRs on openSUSE jobs. Sorry if I added you as reviewer too early.